php/api.php

```diff
@@ -5,12 +5,6 @@
 # @copyright	2015 by Tobias Reich
 ###
 
-@ini_set('max_execution_time', '200');
```

Comment:
Good idea. Lets add it to the [.htaccess](https://github.com/electerious/Lychee/blob/master/.htaccess).